docs/content/guides/getting-started/react-redux.md

```diff
@@ -1,14 +1,18 @@
 ---
-title: 'Redux'
-metaTitle: 'Redux - Guide - Handsontable Documentation'
+title: 'Integration with Redux'
```

Comment:
Do you think the position of this page in the menu is correct now? I think it can stay in the "Getting started" group, but could be moved to the bottom of it.

This file controls the menu links: https://github.com/handsontable/handsontable/blob/809531257acd997d21e882f3a79c5c707fc9d626/docs/content/guides/sidebar.js#L6

Reply:
Will change